backend/oasst_backend/tree_manager.py

```diff
@@ -366,8 +367,14 @@ def determine_task_availability(self, lang: str) -> dict[protocol_schema.TaskReq
             lang = "en"
             logger.warning("Task availability request without lang tag received, assuming lang='en'.")
 
+        init_prompt_disabled_langs = os.getenv("INIT_PROMPT_DISABLED_LANGS", "none")
```

Comment:
Could you move this to the [TreeManagerConfiguration](https://github.com/LAION-AI/Open-Assistant/blob/main/backend/oasst_backend/config.py)? The setting then still can be set via env-variable and this setting would be handled in the same way all other TreeManager settings are handled.